Misc/NEWS.d/next/Library/2024-01-14-11-43-31.gh-issue-113878.dmEIN3.rst

```diff
@@ -0,0 +1,2 @@
+Add ``doc`` parameter to :func:`dataclasses.field`, so it can be stored and
+shown as a documentation / metadata.
```

Comment:
This should mention that it's only visible outside of the Field objects when `slots=True`. It should also mention that dataclass-created `__slots__` is changing from a tuple to a dict in order to support this.